resources/carrier/ru/374.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2013 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Manually translated
+# http://en.wikipedia.org/wiki/Telephone_numbers_in_Armenia#Mobile_network_codes
```

Comment:
Please change to "# Translated manually from en/374.txt." to be consistent with https://github.com/googlei18n/libphonenumber/blob/master/resources/geocoding/ru/374.txt - we track sources at the en/ files.